packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -148,7 +148,7 @@
 
 // Add space between adjacent MessageBars.
 .root + .root {
-  margin-bottom: 6px;
+  margin-top: 6px;
```

Comment:
no change file! Also... I think this is the wrong fix. I would argue that we should have no margin on any simple component and leave margins to collection components which can abstract the positioning. E.g. you want to render multiple messagebars? Define a messageBar frame element, which can animate and space the message bars in a stack in the designated area.